### PR TITLE
docs: update changelog

### PR DESCRIPTION
**Pull request by MuyangAmigo**
@microsoft/teamsfx-cli	v1.2.2-rc.2
@microsoft/teamsfx-core	v1.23.0-rc.2
ms-teams-vscode-extension	v4.2.2-rc.2

CLI BREAKING CHANGE:
  

Extension-toolkit BREAKING CHANGE:
 

CLI feat commits:
  

Extension-toolkit feat commits:

### CODE REVIEW

**Review by sffamily — DISMISSED** [review]

**Review by therealjohn — CHANGES_REQUESTED** [review]
Needs more specific information on what bugs were fixed.

**Comment by pierceboggan** [comment]
Agree with @therealjohn!

**Review by XiaofuHuang — DISMISSED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+- Updated pre-requisite checker, applicable app templates and samples to support Node.js 18.
+- Updated comments on Notification bot template for how to send notifications to an individual person.
+- Updated TeamsFx-React SDK to support React 17.
+- Updated TeamsFx SDK, relevant bot app templates and samples to use CloudAdapter.
+- Updated app templates to use TeamsJS SDK v2.7.1.
+
+Bug fixes:
+
+- Fxied an issue where login page will pop up even after signed into Microsoft 365 account from Teams Toolkit.
+- Fixed an issue where it may fail to run ngrok on Windows 11 because of whitespaces in path.
```
Comment (swatDong) [inline packages/vscode-extension/CHANGELOG.md:18]: In my understand the issue is not only for *Windows 11*.
@XiaofuHuang - is that for all *Windows* platforms or all platforms?

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+## 4.2.2 - Feb 7, 2023
+
+Incremental version for Teams Toolkit with multiple bugs fixes and enhancements.
+
+Enhancements:
+
+- Updated pre-requisite checker, applicable app templates and samples to support Node.js 18.
+- Updated comments on Notification bot template for how to send notifications to an individual person.
+- Updated TeamsFx-React SDK to support React 17.
+- Updated TeamsFx SDK, relevant bot app templates and samples to use CloudAdapter.
```
Comment (swatDong) [inline packages/vscode-extension/CHANGELOG.md:12]: Would it be better to add link to *CloudAdapter* (e.g., https://learn.microsoft.com/javascript/api/botbuilder/cloudadapter), or call out it's Bot Framework SDK's CloudAdapter?

**Review by therealjohn — DISMISSED** [review]

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/CHANGELOG.md:18]: > In my understand the issue is not only for _Windows 11_. @XiaofuHuang - is that for all _Windows_ platforms or all platforms?

It is for all platform, include mac and linux.

Comment (MuyangAmigo) [reply packages/vscode-extension/CHANGELOG.md]: Thanks, updated.

File: packages/vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/CHANGELOG.md]: Updated.

**Review by swatDong — APPROVED** [review]

**Pull request merged**